File: C/Iniciante/1036formulabhaskara.c
#include <stdio.h>
#include <math.h>
int main() {
    double A,B,C,delta,r1,r2;
    scanf("%lf%lf%lf",&A,&B,&C);
    delta=pow(B,2)-4.0*A*C;
    if(A!=0){
        if(delta>=0){
            r1=(-B+sqrt(delta))/(2.0*A);
            r2=(-B-sqrt(delta))/(2.0*A);
            printf("R1 = %.5lf\nR2 = %.5lf\n",r1,r2);
        }else{
            printf("Impossivel calcular\n");
        }
    }else{
        printf("Impossivel calcular\n");
    }
    return 0;
}

File: Python/iniciante/1036formulabhaskara.py
import math as m
linha = input().split(" ")
a, b, c = linha
a = float(a)
b = float(b)
c = float(c)
delta = b ** 2 - 4 * a * c
if a != 0:
    if delta >= 0:
        r1 = (-b + m.sqrt(delta)) / (2 * a)
        r2 = (-b - m.sqrt(delta)) / (2 * a)
        print("R1 = %.5lf\nR2 = %.5lf" % (r1, r2))
    else:
        print("Impossivel calcular")
else:
    print("Impossivel calcular")
